Assets/_Application/Layers/LayerTypes/HierarchicalObject/HierarchicalObjectLayerGameObject.cs

```diff
@@ -333,23 +333,6 @@ public override void OnProxyTransformParentChanged()
             toggleScatterPropertySectionInstantiator.PropertySection.TogglePropertyToggle();
         }
 
-        public static ObjectScatterLayerGameObject ConvertToScatterLayer(HierarchicalObjectLayerGameObject objectLayerGameObject)
-        {
-            // TODO: Use LayerSpawner or App.Layers to replace the gameobject
-            var scatterPrefab = ProjectData.Current.PrefabLibrary.GetPrefabById(ObjectScatterLayerGameObject.ScatterBasePrefabID);
-            var scatterLayer = Instantiate(scatterPrefab) as ObjectScatterLayerGameObject;
-            scatterLayer.Name = objectLayerGameObject.Name + "_Scatter";
-            scatterLayer.Initialize(objectLayerGameObject, objectLayerGameObject.LayerData.ParentLayer as PolygonSelectionLayer);
-            for (var i = objectLayerGameObject.LayerData.ChildrenLayers.Count - 1; i >= 0; i--) //go in reverse to avoid a collectionWasModifiedError
-            {
-                var child = objectLayerGameObject.LayerData.ChildrenLayers[i];
-                child.SetParent(scatterLayer.LayerData, 0);
```

Comment:
Is dit reparenten niet nodig? Want ik ziet dat in de nieuwe code niet meer terugkomen